core/input/keyboard.js

```diff
@@ -306,25 +304,23 @@ Keyboard.prototype = {
             return;
         }
 
-        let target = this._target;
-        let downList = this._keyDownList;
         ['AltLeft', 'AltRight'].forEach(function (code) {
-            if (!(code in downList)) {
+            if (!(code in this._keyDownList)) {
```

Comment:
`this` points to something different here, so this does not work.

Reply:
True. That won't be a problem when moving to arrow functions but it's now.